readme.md

```diff
@@ -4,6 +4,13 @@ Easily identify hot shard and hot key on your Kinesis data streams.
 ## About
 The Amazon Kinesis Hot Shard Advisor is a CLI tool that simplifies identifying whether you have hot key or hot shard issues on your Kinesis data streams. The tool can also identify whether you are hitting the shard level throughput limit per-second basis.
 
+## Why KHS?
+An Amazon Kinesis Data Stream consists of one or more shards. Records with the same partition key is always written to the same shard. Each shard has a [maximum throughput](https://aws.amazon.com/kinesis/data-streams/faqs/#:~:text=A%20shard%20has%20a%20sequence,2%20MB%2Fsecond%20for%20reads.) it can support. When a producer application attempts to put records at a higher rate than the maximum throughput available, they receive `WriteThroughputExceeded` error. This is commonly known as the **hot shard problem**.   
+
+When you experience a hot shard problem, you will notice an increase of `WriteProvisionedThroughputExceeded` metric in CloudWatch. However, for bursty workloads you may not notice the corresponding data in `IncomingRecords` and `IncomingBytes` metrics due to per minute aggregation. You can get the visibility to bursty writes with khs because it reads the records from your stream and aggregate the record size per second based on `ApproximateArrivalTimestamp`.
```

Comment:
all looks good to me. Just one thing - can you add a bit more on "higher rate" so we get a clarity here- i.e "A single shard can ingest up to 1 MB of data per second (including partition keys) "...